src/coreclr/src/System.Private.CoreLib/System.Private.CoreLib.csproj

```diff
@@ -13,7 +13,7 @@
     <DebugSymbols>true</DebugSymbols>
 
     <!-- Force System.Private.CoreLib.dll into a special IL output directory -->
-    <OutputPath>$(BinDir)IL/</OutputPath>
+    <OutputPath>$(BinDir)/IL/</OutputPath>
```

Comment:
The rest of this project file seems to prefer \ over /

Reply:
It looks like it's split across items and properties. Items use `\` whereas properties in this file tend to use `/`.